src/theme/components/customTable.ts

```diff
@@ -1,38 +1,82 @@
 import { useColorModeValue } from "@chakra-ui/react";
+import { TableBodyTextSizes } from "../../components/Table/Table";
 
 interface BaseStyleProps {
   columnHeadersBackgroundColor?: string;
   columnHeadersTextColor?: string;
+  tableTextSize?: TableBodyTextSizes;
+  isScrollable?: boolean;
   showRowDividers?: boolean;
   useRowHeaders?: boolean;
 }
 
 const CellBorderColorStyles = () => {
-  const colorValues = useColorModeValue(
-    { base: "ui.gray.light-cool", md: "ui.border.default" },
-    { base: "ui.gray.xx-dark", md: "dark.ui.border.default" }
-  );
+  const colorValues = useColorModeValue("ui.gray.semi-medium", {
+    base: "ui.gray.xx-dark",
+    md: "dark.ui.border.default",
+  });
   return colorValues;
 };
+export const fixedColumnStyles = (
+  isScrollable = false,
+  useRowHeaders = false
+) => ({
+  backgroundColor: useRowHeaders ? "ui.gray.x-light-cool" : "null",
+  borderRight: useRowHeaders
+    ? isScrollable
+      ? "1px solid var(--nypl-colors-ui-gray-semi-medium)"
+      : {
+          base: undefined,
+          md: "1px solid var(--nypl-colors-ui-gray-semi-medium)",
+        }
+    : undefined,
+  left: useRowHeaders
+    ? isScrollable
+      ? { base: undefined, md: "0" }
+      : undefined
+    : undefined,
+  position: useRowHeaders ? "sticky" : undefined,
+  zIndex: "5",
+  _dark: {
+    backgroundColor: useRowHeaders ? "dark.ui.bg.default" : undefined,
+    borderRight: useRowHeaders
+      ? "2px solid var(--nypl-colors-dark-ui-border-default)"
+      : undefined,
+  },
+});
+
+const tableTextSizes = (textSizeValue: string = "body1") => ({
+  columnHeading:
+    textSizeValue === "body2"
+      ? "desktop.caption.caption1"
+      : "desktop.body.body2",
+  tableBody:
+    textSizeValue === "body2" ? "desktop.body.body2" : "desktop.body.body1",
+});
 
 export const baseTRStyles = (
   columnHeadersBackgroundColor = "",
+  isScrollable = false,
   showRowDividers = false,
   useRowHeaders = false
 ) => ({
-  borderBottom: { base: "2px solid", md: 0 },
+  borderBottom: isScrollable ? "0" : { base: "2px solid", md: "0" },
```

Comment:
Doesn't make a difference in terms of functionality, but is there a reason these 0 's are being passed as strings?

Reply:
Nope.  I just removed the quotes.  All good.